### PR TITLE
VSTS-283 Requests have timeout

### PR DESCRIPTION
**Pull request by alexandre-holzhey-sonarsource**
Please review our [contribution guidelines](https://github.com/SonarSource/sonar-scanner-vsts/blob/master/docs/contributing.md).

And please ensure your pull request adheres to the following guidelines: 

- [ ] Please explain your motives to contribute this change: what problem you are trying to fix, what improvement you are trying to make
- [ ] Use the following formatting style: [SonarSource/sonar-developer-toolset](https://github.com/SonarSource/sonar-developer-toolset#code-style)
- [ ] Provide a unit test for any code you changed
- [ ] If there is a [JIRA](https://jira.sonarsource.com/browse/VSTS) ticket available, please make your commits and pull request start with the ticket ID (VSTS-XXXX)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
         "prettier": "2.4.1",
         "request": "2.85.0",
         "semver": "5.5.0",
-        "sonarqube-scanner": "2.8.1",
+        "sonarqube-scanner": "3.0.0",
```
Comment (mickael-caro-sonarsource) [inline package-lock.json:53]: Was all the deletion of this package-lock the result of upgrading the scanner here ?

File: common/ts/helpers/__tests__/request-test.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import Endpoint, { EndpointType } from "../../sonarqube/Endpoint";
```
Comment (mickael-caro-sonarsource) [inline common/ts/helpers/__tests__/request-test.ts:1]: You can focus on commonv5 only (it is used only by SQv5 and SonarCloud extensions).

File: package-lock.json (hunk 1 above)
Comment (alexandre-holzhey-sonarsource) [reply package-lock.json]: I don't have any intentions to upgrade packages and i did not add a new one, no idea why this file was updated. I reverted my changes here!

**Comment by alexandre-holzhey-sonarsource** [comment]
According to request [documentation](https://www.npmjs.com/package/request#timeouts), we should receive a

**Comment by alexandre-holzhey-sonarsource** [comment]
Then i published an updated test extension with a bigger timeout (60 seconds), which finished fine as we can see in the [job logs](https://dev.azure.com/alexandreholzhey/ThreeJS/_build/results?buildId=43&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9) and SonarCloud [overview page](https://sonarcloud.io/project/overview?id=alexandreholzhey_ThreeJS%27).

**Review by mickael-caro-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**